<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VerificadorRutaL2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblLoading">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblStatus">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCopy">
        <property name="maximumSize">
         <size>
          <width>125</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Copiar ruta</string>
        </property>
        <property name="icon">
         <iconset resource="resourses.qrc">
          <normaloff>:/copy.png</normaloff>:/copy.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="8">
       <widget class="QPushButton" name="btnIniciar">
        <property name="text">
         <string>Iniciar</string>
        </property>
        <property name="icon">
         <iconset resource="resourses.qrc">
          <normaloff>:/start.png</normaloff>:/start.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QPushButton" name="btnDetener">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Detener</string>
        </property>
        <property name="icon">
         <iconset resource="resourses.qrc">
          <normaloff>:/stop.png</normaloff>:/stop.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>MAC:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLineEdit" name="ledPWD">
        <property name="inputMask">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="6" colspan="2">
       <widget class="QLineEdit" name="ledMAC"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Equipo:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="ledPE">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;IP del equipo desde donde comenzar a seguir la MAC.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="ledGW">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;GW a utilizar si la conexi√≥n a la IP del equipo falla.&lt;/p&gt;&lt;p&gt;Para el caso de los equipos en SV que solo se les llega por ROM-NOC-GTW-01.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>GW:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Usuario:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QLineEdit" name="ledUsuario"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="lytGraphData">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ledPE</tabstop>
  <tabstop>ledGW</tabstop>
  <tabstop>ledMAC</tabstop>
  <tabstop>ledUsuario</tabstop>
  <tabstop>ledPWD</tabstop>
  <tabstop>btnIniciar</tabstop>
  <tabstop>btnDetener</tabstop>
 </tabstops>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ledPWD</sender>
   <signal>returnPressed()</signal>
   <receiver>btnIniciar</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
